addons/web/static/src/views/kanban/kanban_renderer.js

```diff
@@ -475,7 +481,12 @@ export class KanbanRenderer extends Component {
             }
             const refId = previous ? previous.dataset.id : null;
             const targetGroupId = parent?.dataset.id;
-            await this.props.list.moveRecord(dataRecordId, dataGroupId, refId, targetGroupId);
+            try {
+                await this.props.list.moveRecord(dataRecordId, dataGroupId, refId, targetGroupId);
+            } catch (error) {
+                this.toggleProcessing(dataRecordId, false);
+                throw error;
+            }
```

Comment:
```suggestion
            try {
                await this.props.list.moveRecord(dataRecordId, dataGroupId, refId, targetGroupId);
            } finally {
                this.toggleProcessing(dataRecordId, false);
            }
```
(and remove the call below)